### DISCUSSION TITLE
Hide data-maska-value attr

### DISCUSSION THREAD

**Discussion by Ozzmosis94** [beholdr/maska#143] [Q&A]
Can we hide data-maska-value attr when maska is disabled?

**Comment by beholdr** [comment]
You mean to remove `data-maska-value` attribute when mask = null or empty string?

**Reply by Ozzmosis94** [reply]
Yes.

**Comment by Ozzmosis94** [comment]
Most likely the best solution would be not to use an instance of the mask class if mask = null?

**Reply by beholdr** [reply]
Could you please tell what's a case for this?

**Reply by Ozzmosis94** [reply]
For example, if input type = 'password'.
Гser can see the hidden password in devtools.

**Reply by beholdr** [reply]
In devtools you can change type='password' to type='text' and see value as well.
